CHANGELOG.md

```diff
@@ -23,7 +23,9 @@ streamlined to use Sphinx from now on.
  - use C++11 `using` instead of `typedef`
  - removed `Config` suffix in file names #1965
  - `gasConfig` is now `density`
- - `speciesDefinition`: simplified `Particles<>` interface #1711 #1942
+ - `speciesDefinition`:
+   - simplified `Particles<>` interface #1711 #1942
+   - `ionizer< ... >` became a sequence of `ionizers< ... >` #1999
```

Comment:
QUESTION:
@ax3l This is not a bug fix but a feature - should this be back-ported? (Or is this required to add other fixes?)